GTMAppAuthSwift/Sources/GTMKeychain.swift

```diff
@@ -15,9 +15,345 @@
  */
 
 import Foundation
+import Security
 
-/// Utility for saving and loading data to the keychain.
-@objc
-public class GTMKeychain: NSObject {
+/// An internal utility for saving and loading data to the keychain.
+final class GTMKeychain {
+  private var keychainHelper: KeychainHelper
 
+  /// An initializer to create an instance of this keychain wrapper with a given helper.
+  /// - parameter keychainHelper: An instance conforming to `KeychainHelper`.
+  /// - note: This initializer is here to help with testing.
+  init(keychainHelper: KeychainHelper? = nil) {
+    if let helper = keychainHelper {
+      self.keychainHelper = helper
+    } else {
+      self.keychainHelper = KeychainWrapper()
+    }
+  }
+
+  /// Saves the password `String` to the keychain with the given identifier.
+  /// - parameter password: The `String` password.
+  /// - parameter name: The name for the Keychain item.
+  /// - throws: An instance of `KeychainWrapper.Error`.
+  func save(password: String, forName name: String) throws {
+    try savePasswordToKeychain(
+      password: password,
+      name: name
+    )
+  }
+
+  /// Saves the password `String` to the keychain with the given identifier.
+  /// - parameter password: The `String` password.
+  /// - parameter name: The name for the Keychain item.
+  /// - parameter usingDataProtectionKeychain: A `Bool` indicating whether to
+  /// use the data protection keychain on macOS 10.15.
```

Comment:
Should we update for 100 char line length throughout this PR?

Reply:
Yes. Found a bunch throughout.